GreenhouseNode/GreenhouseNode/controllers/humidityD.c

```diff
@@ -0,0 +1,18 @@
+/*
```

Comment:
Instead of having these files postfixed with a `D`, I think we should use ADT (abstract data types). You can check out "Simple ADT Example in C.7z" from ESW lecture "6. Design to Code part 2".
But something like this in eg. the `humidity.c` file
```
typedef struct humuditySensor{
  int_8t id;
  int_16t value;
} humuditySensor;
```
and then this in the header file
```
typedef struct humuditySensor* humuditySensor_t;
```